code/_onclick/hud/hud.dm

```diff
@@ -117,6 +117,8 @@ GLOBAL_LIST_INIT(available_ui_styles, list(
 
 	owner.overlay_fullscreen("see_through_darkness", /atom/movable/screen/fullscreen/see_through_darkness)
 
+	AddComponent(/datum/component/zparallax, owner.client)
```

Comment:
What if you leave and come back? Does /datum/hud get made again?

Reply:
testing

Reply:
fuck